ios/build.sh

```diff
@@ -20,18 +22,22 @@ display_usage ( ) {
 exit 1
 }
 
-fetch ( ) {
-    name="${2##*/}"  # Extract just the filename.
-    echo "Downloading $name"
-    rm $2          2> /dev/null
-    curl -s $1 -o $2
+set_version ( ) {
+  PRODUCT_PATH=$1
+
+  if [ $BUILD_NUMBER ]; then
+    if [ $2 ]; then  # $2 = product name.
+      echo "Setting version numbers in $2 to $BUILD_NUMBER."
+    fi
+    /usr/libexec/Plistbuddy -c "Set CFBundleVersion $BUILD_NUMBER" "$PRODUCT_PATH/Info.plist"
+    /usr/libexec/Plistbuddy -c "Set CFBundleShortVersionString $BUILD_NUMBER" "$PRODUCT_PATH/Info.plist"
+  fi
 }
 
 KMEI_RESOURCES=engine/KMEI/KeymanEngine/resources
 KMEI_BUILD_PATH=engine/KMEI/build
 BUNDLE_PATH=$KMEI_RESOURCES/Keyman.bundle/contents/resources
-APP_FRAMEWORK_PATH=keyman/Keyman/KeymanEngine.framework
-APP_BUILD_PATH=keyman/Keyman/build/
+APP_BUILD_PATH=keyman/Keyman/build
```

Comment:
Since both Keyman and KeymanEngine will be built here, maybe we should move this dir to `ios/build` instead of in the Keyman project directory. And then have `KMEI_BUILD_PATH` point here as well.

Reply:
In fact, KeymanEngine can be built through the workspace rather than the project, in the same way as Keyman, so then hopefully build intermediates will be reused.